compiler/packages/react-mcp-server/src/index.ts

```diff
@@ -353,6 +354,50 @@ Server Components - Shift data-heavy logic to the server whenever possible. Brea
   ],
 }));
 
+server.tool('review-react-runtime', 'Review the runtime of the code and get performance data to evaluate the proposed solution', {
+  text: z.string(),
+},
+async ({text}) => {
+  try {
+    const performanceResults = await measurePerformance(text);
+
+    const formattedResults = `
+# React Component Performance Results
+
+## Render Time
+${performanceResults.renderTime.toFixed(2)}ms
+
+## Web Vitals
+- Cumulative Layout Shift (CLS): ${performanceResults.webVitals.cls?.value || 'N/A'}
+- Largest Contentful Paint (LCP): ${performanceResults.webVitals.lcp?.value || 'N/A'}ms
+- Interaction to Next Paint (INP): ${performanceResults.webVitals.inp?.value || 'N/A'}ms
+- First Input Delay (FID): ${performanceResults.webVitals.fid?.value || 'N/A'}ms
+- Time to First Byte (TTFB): ${performanceResults.webVitals.ttfb?.value || 'N/A'}ms
+
+These metrics can help you evaluate the performance of your React component. Lower values generally indicate better performance.
```

Comment:
This is a nice touch. Could you try asking it to run this tool multiple times to see if it improves on these metrics?

One more thing we might want to do is to run several iterations of the test in a single tool call to reduce noise. Just using the mean for now seems like a good start.